src/index.html

```diff
@@ -4,9 +4,329 @@
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#gadgets">Gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#tablets">Tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#photo">Photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a class="nav__link" href="#video">Video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="imac-27"
+          class="card__image"
+        >
+
+        <div class="card__description">
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <h3 class="card__subtitle">Product code: 195434</h3>
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <span class="card__reviews card__reviews--text">Reviews: 5</span>
+          </div>
+          <div class="card__price">
+            <span class="card__price-text">Price:</span>
+            <span class="card__price-value">$2,199</span>
+          </div>
+          <a
+            href="#"
+            class="card__btn"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+      <div class="card" data-qa="card">
+        <img
+          src="images/imac.jpeg"
+          alt="imac-27"
+          class="card__image"
+        >
+
+        <div class="card__description">
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <h3 class="card__subtitle">Product code: 195434</h3>
+          <div class="card__reviews">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <span class="card__reviews card__reviews--text">Reviews: 5</span>
+          </div>
+          <div class="card__price">
+            <span class="card__price-text">Price:</span>
+            <span class="card__price-value">$2,199</span>
+          </div>
+          <a
+            href="#"
+            class="card__btn"
+            data-qa="hover"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+      <div class="card" data-qa="card">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="card" data-qa="card">
```